applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.h

```diff
@@ -513,6 +518,16 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) EmbeddedSkinVisualizationProcess :
      */
     void RemoveVisualizationProperties();
 
+    template<const bool IsParallel>
```

Comment:
```suggestion
    template<const bool IsDistributed>
```

this is what we usually use, IsParallel is ambiguous

Reply:
OK. Changing it.